cavern/src/main/java/org/opencadc/cavern/CavernInitAction.java

```diff
@@ -103,6 +106,7 @@ public CavernInitAction() {
 
     @Override
     public void doInit() {
+        initIdentityManager();
```

Comment:
`initIdentityManager` needs to be called after `initNodePersistence` because it needs a value of `jndiNodePersistence`